apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage.tsx

```diff
@@ -263,270 +268,313 @@ export const ViewPoolCandidate = ({
       ? snapshotCandidate.pool.classifications[0]?.group
       : "";
 
-    mainContent = (
-      <>
-        {subTitle}
-        <TableOfContents.Section id={sections.minExperience.id}>
-          <TableOfContents.Heading
-            as="h4"
-            size="h5"
-            data-h2-margin="base(x2 0 x.5 0)"
-          >
-            {sections.minExperience.title}
-          </TableOfContents.Heading>
-          <p data-h2-margin="base(x1, 0)">
-            {intl.formatMessage(
-              {
-                defaultMessage:
-                  "Requirement selection: <strong>{educationRequirementOption}</strong>.",
-                id: "J3Ud6R",
-                description:
-                  "Application snapshot minimum experience section description",
-              },
-              {
-                educationRequirementOption: intl.formatMessage(
-                  snapshotCandidate?.educationRequirementOption
-                    ? getEducationRequirementOption(
-                        snapshotCandidate.educationRequirementOption,
-                        classificationGroup,
-                      )
-                    : commonMessages.notAvailable,
-                ),
-              },
-            )}
-          </p>
-          {snapshotCandidate?.educationRequirementExperiences?.length ? (
-            <>
-              <p>
-                {intl.formatMessage({
+    if (features.recordOfDecision) {
+      mainContent = (
+        <>
+          {/**
+           * TODO: Remove `ApplicationStatusForm` with record of decision flag (#8415)
+           *
+           * This is here to keep tests passing
+           */}
+          <ApplicationStatusForm id={poolCandidate.id} />
+          <ApplicationInformation
+            snapshot={parsedSnapshot}
+            application={poolCandidate}
+          />
+          <div data-h2-margin="base(x2 0)">
+            <Accordion.Root type="single" mode="card" collapsible>
+              <Accordion.Item value="otherRecruitments">
+                <Accordion.Trigger>
+                  {intl.formatMessage({
+                    defaultMessage: "Other recruitments",
+                    id: "kZs3Nk",
+                    description:
+                      "Heading for table of a users other applications and recruitments",
+                  })}
+                </Accordion.Trigger>
+                <Accordion.Content>
+                  <PoolStatusTable user={poolCandidate.user} pools={pools} />
+                </Accordion.Content>
```

Comment:
This being nestled in the conditional on line 255
`  if (showRichSnapshot) {`
Results in a page where if there is no snapshot, you cannot see the pool status table. 
Applications with no snapshot are most easily created by adding a user to a pool on the user's profile view, so are probably not rare.

Reply:
To illustrate 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/11e4ac63-028a-424e-91c2-9e7ab87e32c5)